src/bin/dual-iir.rs

```diff
@@ -202,14 +220,20 @@ mod app {
         let clock = SystemTimer::new(|| monotonics::now().ticks() as u32);
 
         // Configure the microcontroller
-        let (stabilizer, _pounder) = hardware::setup::setup(
+        let (stabilizer, pounder) = hardware::setup::setup(
             c.core,
             c.device,
             clock,
             BATCH_SIZE,
             SAMPLE_TICKS,
         );
 
+        let (pounder_devices, dds_output) = if let Some(pounder_dev) = pounder {
```

Comment:
It was to separate the DDS modules as a local resource instead of global resource. DDS is only needed when settings are updated.
Though it is not like everything in the `PounderDevices` struct is needed in multiple tasks.